sdk/core/azure-core/CHANGELOG.md

```diff
@@ -16,12 +20,12 @@
   - `GetMinorVersion`.
 - Renamed `Azure::Nullable<T>::GetValue()` to `Value()`.
 - Changes to `Azure::Core::Context`:
-  - Renamed `Get()` to `GetValue()`.
+  - Removed `Get()` and `HasKey()`.
```

Comment:
"Removed `Get()` and `HasKey()` in favor of a new method `TryGetValue()`"? So, it is actionable as you read it, what to replace it with.